### PR TITLE
Fix API instrumentation

### PR DESCRIPTION
**Pull request by joshblum**
depends on https://github.com/keybase/go-framed-msgpack-rpc/pull/182

### CODE REVIEW

File: go/avatars/fullcaching.go [hunk 1]
```diff
@@ -257,7 +257,7 @@ func (c *FullCachingSource) populateCacheWorker(m libkb.MetaContext) {
 		found, ent, err := c.diskLRU.Get(m.Ctx(), m.G(), key)
 		if err != nil {
 			c.debug(m, "populateCacheWorker: failed to read previous entry in LRU: %s", err)
-			err := libkb.DiscardAndCloseBody(resp)
+			_, err := libkb.DiscardAndCloseBody(resp)
```
Comment (mlsteele) [inline go/avatars/fullcaching.go]: This looks unmetered. Is that true or intentional? I'm not familiar with this, arg.url could be local.

File: go/service/config.go [hunk 2]
```diff
@@ -581,19 +581,11 @@ func (h ConfigHandler) UpdateLastLoggedInAndServerConfig(
...
-	resp, finisher, err := h.G().API.GetResp(mctx, arg)
-	defer finisher()
-	if err != nil {
-		return err
-	}
-	userFeatureusBytes, err := ioutil.ReadAll(resp.Body)
-	if err != nil {
-		return err
-	}
-	jw, err := jsonw.Unmarshal(userFeatureusBytes)
```
Comment (joshblum) [inline go/service/config.go]: cc @songgao, don't think we need to stream the response here, can just let the api library do the json unmarshal

File: go/client/cmd_ping.go [hunk 3]
```diff
@@ -97,7 +97,7 @@ func (t *pingGregorTransport) Dial(context.Context) (rpc.Transporter, error) {
 		return nil, err
 	}
 	t.stagedTransport = rpc.NewTransport(t.conn, nil,
-		rpc.NewNetworkInstrumenter(t.G().NetworkInstrumenterStorage),
```
Comment (joshblum) [inline go/client/cmd_ping.go]: pass nil?

File: go/avatars/fullcaching.go (hunk 1 above)
Comment (joshblum) [reply go/avatars/fullcaching.go]: Ah good catch, missed a few of `libkb.ProxyHTTPGet`

Comment (joshblum) [reply go/avatars/fullcaching.go]: will update this in a subsequent PR, a larger change than expected

**Comment by joshblum** [comment]
merging over known flake

**Pull request merged**